test/features/bootstrap/Quickstart/Bundle/AppBundle/Features/Context/HomepageContext.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Quickstart\Bundle\AppBundle\Features\Context;
+
+use SensioLabs\Behat\PageObjectExtension\Context\PageObjectContext;
+
+class HomepageContext extends PageObjectContext
```

Comment:
Ignore me! i didn't read this part of the docs, the other way to do it :smile: http://behat-page-object-extension.readthedocs.org/en/latest/guide/working_with_page_objects.html#using-the-page-object-factory